src/components/v5/common/Table/types.ts

```diff
@@ -0,0 +1,19 @@
+import { ColumnDef } from '@tanstack/react-table';
+
+export interface TableProps<T> {
+  className?: string;
+  tableTitle?: React.ReactNode;
+  columns: ColumnDef<T>[];
+  fields: T[];
+  burgerColumn: ColumnDef<T, any>[];
+  setSelectedRowId: React.Dispatch<React.SetStateAction<string | undefined>>;
+}
+
+export interface BurgerMenuProps {
```

Comment:
Please don't mix interfaces. If this one is used in other directory, then it should be added there.

Reply:
it will remover  it after @CzarekDryl PR with burger will be merged

Reply:
I think that it can be removed now, without waiting for Czarek's changes